### ISSUE TITLE
background image should tile in original resolution by default

### ISSUE THREAD

**Issue by lukaprincic** [funmusicplace/mirlo#835]
when I upload a background image I expect it to be tiled in it's original resolution, not stretched. 

I don't think anyone likes the stretched image in the background. If I want a big heavy image in the background, I'll upload it and maybe I can turn off tiling (an option/toggle in the design section?), but I would rarely want to have it stretched. To me it is always ugly.

**Comment by samnabi** [comment]
I feel like the intended effect should really be up to the artist. If I upload a 1280px by 720px image and somebody views it on a 4K screen, I probably want it to stretch instead of tile.

If someone uploads a small background image and it doesn't stretch or tile, it could just get hidden behind the main content block.

**Comment by lukaprincic** [comment]
I agree with the fact that whoever is customizing the page appearance should have an option to decide. But in general I would say that the background is meant to be a texture not an image that visitor should see in full, so that's why I propagate for tiling in original resolution. Bandcamp has this behaviour but also great range of choices, as evident in the screenshot below...

![image](https://github.com/user-attachments/assets/bb7c3019-9b92-4301-b3f9-1ddc62d7ffb7)
